R/labels.R

```diff
@@ -93,13 +100,16 @@ uses_labels <- function(p, ...) {
 
   labels <- get_labels(p, names(args))
 
-  null_expected <- lengths(args) == 0
-  nulls_match   <- all(lengths(labels[null_expected]) == 0)
-  strings_match <- all(
-    mapply(identical, args[!null_expected], labels[!null_expected])
+  result <- logical(length(args))
+
+  null_expected          <- lengths(args) == 0
+  result[null_expected]  <- lengths(labels[null_expected]) == 0
+  result[!null_expected] <- as.logical(
+    Map(identical, args[!null_expected], labels[!null_expected])
```

Comment:
Should we import purrr? Maybe not just for this call, but there are more of these lying around we might want to swap them out.

Reply:
`purrr` is a `gradethis` dependency anyway, so I don't see any downside